cartoframes/io/managers/context_manager.py

```diff
@@ -428,6 +449,11 @@ def _cartodbfy_query(table_name, schema):
         schema=schema, table_name=table_name)
 
 
+def _regenerate_table_query(table_name, schema):
+    return 'SELECT CDB_RegenerateTable(\'{schema}.{table_name}\'::regclass)'.format(
```

Comment:
Instead of escaping the quotes we could double quote the external string to improve readability:
```
return "SELECT CDB_RegenerateTable('{schema}.{table_name}'::regclass)".format(
    schema=schema, table_name=table_name)
```